src/server/routes/api/document/indra.js

```diff
@@ -19,10 +21,23 @@ const BASE_MODIFICATION_TYPES = ['Sumoylation', 'Desumoylation', 'Hydroxylation'
 const TRANSCRIPTION_TRANSLATION_TYPES = ['IncreaseAmount', 'DecreaseAmount'];
 const BINDING_TYPES = ['Complex'];
 
-const getDocuments = pairs => {
-  const getForPair = pair => {
-    let agent0 = pair[0].toUpperCase();
-    let agent1 = pair[1].toUpperCase();
+const getDocuments = templates => {
+  const getForIntn = intnTemplate => {
+    const getAgent = t => {
+      let agent;
+
+      if ( t.xref ) {
+        agent = t.xref.id + '@' + t.xref.namespace;
```

Comment:
Beware: HGNC database records have IDs that look like `HGNC:12345` but INDRA wants `12345`

Reply:
Would using what is remaining after ":" save any possible cases?

Reply:
Can you clarify what you mean by 'save'?

Reply:
I was meaning: would trimming the text after ":" enough for any case?

Reply:
Yes that's fine for HGNC at least.